### DISCUSSION TITLE
Set up resource group for serviceowner in their subscription

### DISCUSSION THREAD

**Discussion by bengtfredh** [Altinn/altinn-platform#755] [Ideas]
We have som outstanding issues related to i.e creating alerts, and we are thinking about creating key vault for each app. If we want service owners to create alerts from Azure we need to create a resource group where they have permissions to create alerts. If we want to have key vault for each app, it will be more easy to administrate if we put this key vaults in same resource group. Like this we separate the resources service owners have access to from other resources. We can probably think of more resources service owners should have more access to.

List of resources that can be put i serviceowner-rg:
- alerts
- key vault
- log?

**Comment by tjololo** [comment]
solved by leveraging alerts and dashboards in grafana
